app/controllers/votes_controller.rb

```diff
@@ -27,5 +25,9 @@ def update_vote(new_value)
       @vote = current_user.votes.create(value: new_value, post: @post)
     end
 
+    respond_to do |format|
+      format.html
+      format.js
+    end
   end
```

Comment:
One would probably keep this code inside `up_vote` and `down_vote`, *duplicated*, for it's not `update_vote`'s responsibility to *respond* to the HTTP request. It really is an action's duty.

Duplication could be reduced by [using `respond_with`](https://github.com/plataformatec/responders), a feature provided by an external gem (it used to be a Rails built-in, but it has been extracted for it [has its own issues](http://stackoverflow.com/questions/25998437/why-is-respond-with-being-removed-from-rails-4-2-into-its-own-gem)).